### DISCUSSION TITLE
404 during benchmak

### DISCUSSION THREAD

**Discussion by ghost** [pocketbase/pocketbase#448] [Q&A]
Hi,

I benchmarked the performance of pocketbase api with bombardier.
The benchmark was made on a simple collection holding one record.
Benchmarking at  >70 requests per second started to trigger 404 errors.

Are these errors because of `github.com/pocketbase/dbx` or it's something else?

**Comment by ganigeorgiev** [comment]
404 is strange response, If something is wrong with the API the most common error would be 400.

If your server has at least 1vCPU and 1GB RAM you should be able to send without any troubles 1000+ **concurrent/simultaneous** rps (for more you may need to increase the `ulimit` depending on your os configuration). My local tests with `hey` reached up to 10 000+ **concurrent/simultaneous** realtime connect + updates on Hetzner CX21 (2vCPU, 4GB ram) before it started swapping.

- How do you run bombardier (with what options) and against what endpoint?

- What type of request you are benchmarking - get, post, patch, delete?

- Why do you suspect `github.com/pocketbase/dbx`?

_I'll move the issue to Q&A discussion._
